Current master is broken when configuring with zfp

when using `-DADIOS2_USE_ZFP=ON` the following error shows up:

```
-- Verifying Fortran/CXX Compiler Compatibility - Success
CMake Error at testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:6 (include):
  include could not find load file:

    ADIOS2Functions
Call Stack (most recent call first):
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:165 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:168 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:171 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:165 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:168 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:171 (set_property):
  set_property given TEST names that do not exist:

   HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


-- Found MPI: TRUE (found version "3.1") found components:  C 

ADIOS2 build configuration:
  ADIOS Version: 2.4.0.442
  C++ Compiler : GNU 7.4.0 
    /usr/bin/c++

  Fortran Compiler : GNU 7.4.0 
    /usr/bin/f95

  Installation prefix: /home/wgodoy/tmp/adios2
        bin: bin
        lib: lib
    include: include
      cmake: lib/cmake/adios2
     python: lib/python3.6/site-packages

  Features:
    Library Type: shared
    Build Type:   Debug
    Testing: ON
    Build Options:
      Blosc    : ON
      BZip2    : ON
      ZFP      : ON
      SZ       : ON
      MGARD    : ON
      PNG      : ON
      MPI      : ON
      DataMan  : OFF
      Table    : OFF
      SSC      : OFF
      SST      : ON
      DataSpaces: OFF
      ZeroMQ   : ON
      HDF5     : ON
      Python   : ON
      Fortran  : ON
      SysVShMem: ON
      Profiling: ON
      Endian_Reverse: OFF
    RDMA Transport for Staging: Unconfigured

-- Configuring incomplete, errors occurred!
See also "/home/wgodoy/tmp/ADIOS2-build/CMakeFiles/CMakeOutput.log".
See also "/home/wgodoy/tmp/ADIOS2-build/CMakeFiles/CMakeError.log".
```

--- a/testing/adios2/bindings/C/TestBPWriteTypes.cpp
+++ b/testing/adios2/bindings/C/TestBPWriteTypes.cpp
@@ -42,6 +42,7 @@ public:
 
 TEST_F(ADIOS2_C_API, ADIOS2BPWriteTypes)
 {
+    const char fname[] = "ADIOS2_C_API.ADIOS2BPWriteTypes.bp";
     // write
     {
         // IO
@@ -148,8 +149,7 @@ TEST_F(ADIOS2_C_API, ADIOS2BPWriteTypes)
         adios2_variable *varR32 = adios2_inquire_variable(ioH, "varR32");
         adios2_variable *varR64 = adios2_inquire_variable(ioH, "varR64");
 
-        adios2_engine *engineH =
-            adios2_open(ioH, "ctypes.bp", adios2_mode_write);
+        adios2_engine *engineH = adios2_open(ioH, fname, adios2_mode_write);
 
         adios2_put(engineH, varStr, dataStr, adios2_mode_sync);
         adios2_put(engineH, varI8, data_I8, adios2_mode_deferred);
@@ -184,8 +184,7 @@ TEST_F(ADIOS2_C_API, ADIOS2BPWriteTypes)
 #endif
     {
         adios2_io *ioH = adios2_declare_io(adiosH, "Reader");
-        adios2_engine *engineH =
-            adios2_open(ioH, "ctypes.bp", adios2_mode_read);
+        adios2_engine *engineH = adios2_open(ioH, fname, adios2_mode_read);
 
         size_t steps;
         adios2_steps(&steps, engineH);
@@ -399,6 +398,7 @@ std::string adios2_engine_name_as_string(adios2_engine *engineH)
 
 TEST_F(ADIOS2_C_API_IO, Engine)
 {
+    const char fname[] = "ADIOS2_C_API_IO.engine.bp";
     int ierr;
 
     ierr = adios2_set_engine(ioH, "bpfile");
@@ -407,16 +407,17 @@ TEST_F(ADIOS2_C_API_IO, Engine)
     std::string engine_type = testing::adios2_engine_type_as_string(ioH);
     EXPECT_EQ(engine_type, "bpfile");
 
-    adios2_engine *engineH = adios2_open(ioH, "ctypes.bp", adios2_mode_write);
+    adios2_engine *engineH = adios2_open(ioH, fname, adios2_mode_write);
 
     // FIXME, I'd like to check that the engine type itself is correct, but
     // there's no API to get it
     std::string engine_name = testing::adios2_engine_name_as_string(engineH);
-    EXPECT_EQ(engine_name, "ctypes.bp");
+    EXPECT_EQ(engine_name, fname);
 }
 
 TEST_F(ADIOS2_C_API_IO, EngineDefault)
 {
+    const char fname[] = "ADIOS2_C_API_IO.EngineDefault.bp";
     int ierr;
 
     ierr = adios2_set_engine(ioH, "");
@@ -425,12 +426,12 @@ TEST_F(ADIOS2_C_API_IO, EngineDefault)
     std::string engine_type = testing::adios2_engine_type_as_string(ioH);
     EXPECT_EQ(engine_type, "");
 
-    adios2_engine *engineH = adios2_open(ioH, "ctypes.bp", adios2_mode_write);
+    adios2_engine *engineH = adios2_open(ioH, fname, adios2_mode_write);
 
     // FIXME, I'd like to check that the engine type itself is correct, but
     // there's no API to get it
     std::string engine_name = testing::adios2_engine_name_as_string(engineH);
-    EXPECT_EQ(engine_name, "ctypes.bp");
+    EXPECT_EQ(engine_name, fname);
 }
 
 TEST_F(ADIOS2_C_API_IO, ReturnedStrings)

--- a/testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake
+++ b/testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake
@@ -3,7 +3,7 @@
 # accompanying file Copyright.txt for details.
 #------------------------------------------------------------------------------#
 
-include(ADIOS2Functions)
+include(ADIOSFunctions)
 
 add_test(NAME HeatTransfer.BP3.zfp.Mx1.Write
   COMMAND ${MPIEXEC_EXECUTABLE}

